internal/infrastructure/persistence/sqlite/queries/history.sql

```
@@ -36,14 +43,17 @@ DELETE FROM history WHERE id = ?;
 -- name: DeleteHistoryOlderThan :exec
 DELETE FROM history WHERE last_visited < ?;
 
+-- name: DeleteHistorySince :exec
+DELETE FROM history WHERE last_visited >= ?;
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)